[CI] InstallPluginActionTests class failing

**Build Scans:**
- [elasticsearch-periodic #6524 / openjdk24_entitlements_true_checkpart1_java-matrix](https://gradle-enterprise.elastic.co/s/h3rphtx7d2kp6)
- [elasticsearch-periodic #6524 / openjdk24_entitlements_false_checkpart1_java-matrix](https://gradle-enterprise.elastic.co/s/lfuudimeypc4k)
- [elasticsearch-periodic #6517 / openjdk24_entitlements_true_checkpart1_java-matrix](https://gradle-enterprise.elastic.co/s/uj472qgdkr7f2)
- [elasticsearch-periodic #6517 / openjdk24_entitlements_false_checkpart1_java-matrix](https://gradle-enterprise.elastic.co/s/m57knpovtbdcw)
- [elasticsearch-periodic #6510 / openjdk24_entitlements_false_checkpart1_java-matrix](https://gradle-enterprise.elastic.co/s/kfoyfirn32rqc)
- [elasticsearch-periodic #6510 / openjdk24_entitlements_true_checkpart1_java-matrix](https://gradle-enterprise.elastic.co/s/nnxuoyuvvtgli)

**Reproduction Line:**
```
./gradlew ":distribution:tools:plugin-cli:test" --tests "org.elasticsearch.plugins.cli.InstallPluginActionTests" -Dtests.method="testBatchFlag {p0=com.google.common.jimfs.JimfsFileSystem@14998e21 p1=org.elasticsearch.plugins.cli.InstallPluginActionTests$1Parameter$$Lambda/0x00000000873de7f8@1921994e}" -Dtests.seed=3D91BE5DD255297B -Dtests.jvm.argline="-Des.entitlements.enabled=true" -Dtests.locale=ln-CF -Dtests.timezone=America/Panama -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.plugins.cli.InstallPluginActionTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(org.elasticsearch.plugins.cli.InstallPluginActionTests),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: a string containing "WARNING: plugin requires additional permissions"
     but: was ""
```

**Issue Reasons:**
- [main] 2 consecutive failures in class org.elasticsearch.plugins.cli.InstallPluginActionTests
- [main] 3 consecutive failures in step openjdk24_entitlements_true_checkpart1_java-matrix
- [main] 3 consecutive failures in step openjdk24_entitlements_false_checkpart1_java-matrix
- [main] 6 failures in class org.elasticsearch.plugins.cli.InstallPluginActionTests (3.0% fail rate in 201 executions)
- [main] 3 failures in step openjdk24_entitlements_true_checkpart1_java-matrix (100.0% fail rate in 3 executions)
- [main] 3 failures in step openjdk24_entitlements_false_checkpart1_java-matrix (100.0% fail rate in 3 executions)
- [main] 3 failures in pipeline elasticsearch-periodic (75.0% fail rate in 4 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/distribution/tools/plugin-cli/src/test/java/org/elasticsearch/plugins/cli/InstallPluginActionTests.java
+++ b/distribution/tools/plugin-cli/src/test/java/org/elasticsearch/plugins/cli/InstallPluginActionTests.java
@@ -48,6 +48,7 @@ import org.elasticsearch.core.SuppressForbidden;
 import org.elasticsearch.core.Tuple;
 import org.elasticsearch.env.Environment;
 import org.elasticsearch.env.TestEnvironment;
+import org.elasticsearch.jdk.RuntimeVersionFeature;
 import org.elasticsearch.plugin.scanner.NamedComponentScanner;
 import org.elasticsearch.plugins.Platforms;
 import org.elasticsearch.plugins.PluginDescriptor;
@@ -891,6 +892,7 @@ public class InstallPluginActionTests extends ESTestCase {
     }
 
     public void testBatchFlag() throws Exception {
+        assumeTrue("security policy validation only available with SecurityManager", RuntimeVersionFeature.isSecurityManagerAvailable());
         installPlugin(true);
         assertThat(terminal.getErrorOutput(), containsString("WARNING: plugin requires additional permissions"));
         assertThat(terminal.getOutput(), containsString("-> Downloading"));
@@ -1529,6 +1531,7 @@ public class InstallPluginActionTests extends ESTestCase {
     }
 
     public void testPolicyConfirmation() throws Exception {
+        assumeTrue("security policy parsing only available with SecurityManager", RuntimeVersionFeature.isSecurityManagerAvailable());
         writePluginSecurityPolicy(pluginDir, "getClassLoader", "setFactory");
         InstallablePlugin pluginZip = createPluginZip("fake", pluginDir);
 

